README.md

```diff
@@ -17,7 +19,7 @@ Want to get paid for your contributions to `one-app-ducks`?
 
 * [Features](#-features)
 * [Usage](#-usage)
-* [API](#%EF%B8%8F-api)
+* [API](#%EF%B8%8F-one-app-ducks-api)
```

Comment:
Why did this change?

Reply:
Changed the API title to `One App Ducks API` for web experience purposes

Reply:
Is this something that will be happening across all repos?

Reply:
We will need to add Metadata comments to all Markdowns in order to have correct tags, title and authorship.

Some libraries like Iguazu that only have a single README will require Hide tags added to remove unnecessary Markdown sections like "Contributing" for the web experience.